cirq/sim/wave_function_simulator.py

```diff
@@ -164,27 +165,23 @@ def compute_displays_sweep(
     def compute_amplitudes_sweep(
             self,
             program: Union[circuits.Circuit, schedules.Schedule],
-            bitstrings: np.ndarray,
+            bitstrings: Sequence[int],
             params: study.Sweepable,
             qubit_order: ops.QubitOrderOrList = ops.QubitOrder.DEFAULT,
-    ) -> List[List[complex]]:
+    ) -> Sequence[Sequence[complex]]:
         circuit = (program if isinstance(program, circuits.Circuit) else
                    program.to_circuit())
-
         trial_results = self.simulate_sweep(circuit, params, qubit_order)
 
-        qid_shape = circuit.qid_shape(qubit_order=qubit_order)
-        amplitude_indices = [
-            value.big_endian_digits_to_int(bitstring, base=qid_shape)
-            for bitstring in bitstrings
-        ]
+        if isinstance(bitstrings, tuple):
```

Comment:
What about all the other possible types of sequence? Why is tuple being handled specially?

Reply:
1-dimensional tuples are handled specially by numpy when used as an array index: https://docs.scipy.org/doc/numpy/reference/arrays.indexing.html#advanced-indexing

"Advanced indexing is triggered when the selection object, obj, is a non-tuple sequence object ..."

Reply:
Comment that in the code.

Reply:
Done.

Reply:
Done.